src/search/landmarks/landmark_factory_rpg_exhaust.cc

```diff
@@ -17,7 +17,7 @@ namespace landmarks {
    method with others, don't use it by itself. */
 
 LandmarkFactoryRpgExhaust::LandmarkFactoryRpgExhaust(const Options &opts)
-    : LandmarkFactoryRelaxation(opts) {
+    : only_causal_landmarks(opts.get<bool>("only_causal_landmarks")) {
```

Comment:
As stated earlier, I personally would always call the super constructor. But I don't know if we have any conventions for this.